### ISSUE TITLE
Switch devenv.com to msbuild

### ISSUE THREAD

**Issue by jbarber2016** [avdbrink/VSTS-SSRS-Extention#5]
Creating an issue for devenv.com usage.  Some versions will cause the devenv.com to throw an error for License on Visual Studio.  msbuild is supported, just need to figure out proper build arguements.

**Comment by avdbrink** [comment]
Doesn't msbuild throw the same exception if the license for visual studio is expired?

**Comment by jbarber2016** [comment]
@avdbrink No, because msbuild is a command line that comes with .net sdk.  So, it shouldn't throw the error.  devenv.com is what Visual Studio uses so therefore if the license is expired you'll get a message.
